backend/homework/3/src/main/java/shrishti/example/BillingComponent/BlueCrossBlueShield.java

```diff
@@ -0,0 +1,41 @@
+package shrishti.example.BillingComponent;
+
+
+public class BlueCrossBlueShield implements InsuranceBrand{
+    public double computeMonthlyPremium(HealthInsurancePlan insurancePlan, int age, boolean smoking){
+        double answer = 0.0;
+        if(insurancePlan instanceof PlatinumPlan){
+            if(age>55){
+                answer+=200.0;
+            }
+            if(smoking){
+                answer+=100.0;
+            }
```

Comment:
This could be generalised and used for all insurance plans. Lets separate this into a function.